src/components/hostFamilyCard/HostFamilyCard.tsx

```diff
@@ -6,81 +6,75 @@ import { Link, useNavigate } from "react-router-dom";
+        <CardHeader className="w-full">
+          <img
+            className="w-full object-cover object-center"
+            src={hostFamily.photoUrl || defaultImage}
+            alt={hostFamily.displayName ?? "Family"}
+          />
+        </CardHeader>
 
-                <CardContent className="flex-grow">
-                    <CardContent>
-                        <div className="flex flex-col lg:flex-row sm:justify-between items-center">
-                            <div className="flex gap-2 mb-4 lg:mb-0 flex-col">
-                                <ItemInfo
-                                    nativeLanguage={
-                                        hostFamily.familyLanguage ?? "Langues ?"
-                                    }
-                                    icon={<Flag />}
-                                />
-                                <ItemInfo
-                                    region={hostFamily.region}
-                                    icon={<MapPin />}
-                                />
-                                <ItemInfo
-                                    price={hostFamily.familyDailyRate ?? 0}
-                                    icon={<Euro />}
-                                >
-                                    / jour
-                                </ItemInfo>
-                            </div>
-                            <Button
-                                className="w-full lg:w-fit"
-                                onClick={() => navigate("/not-sale")}
-                            >
-                                Réserver
-                            </Button>
-                        </div>
-                    </CardContent>
-                    <CardContent className="border-t border-t-secondary pt-4">
-                        <CardTitle className="text-secondary text-balance py-2">
-                            Bienvenue chez la famille{" "}
-                            <span className="text-gray-600">
-                                {hostFamily.displayName}
-                            </span>
-                        </CardTitle>
+        <CardContent className="flex-grow">
+          <CardContent>
+            <div className="flex flex-col lg:flex-row sm:justify-between items-center">
+              <div className="flex gap-2 mb-4 lg:mb-0 flex-col">
+                <ItemInfo
+                  nativeLanguage={hostFamily.familyLanguage ?? "Langues ?"}
+                  icon={<Flag />}
+                />
+                <ItemInfo region={hostFamily.region} icon={<MapPin />} />
+                <ItemInfo
+                  price={hostFamily.familyDailyRate ?? 0}
+                  icon={<Euro />}
+                >
+                  / jour
+                </ItemInfo>
+              </div>
+              <Button
+                className="w-full lg:w-fit"
+                onClick={() => navigate("/not-sale")}
```

Comment:
Ici il faut renvoyer sur le fake stripe avec
```
onClick={handleReserve}
``` 
et rajouter le meme code que dans `Family.tsx`

Reply:
Et supprimer `/not-sale`

Reply:
Ok j'ai supprimé NotSale et la route, mais le bouton ne sert à rien car impossible de cliquer dessus vu que toute la card est un lien.
Je l'enlève et on peut cliquer sur RESERVER quand on visualise une famille.

Reply:
Je pense que c'est plutôt intuitif qu'il faille cliquer sur une famille pour avoir plus d'info...